code/modules/antagonists/malf_ai/datum_malf_ai.dm

```diff
@@ -89,21 +89,23 @@
 
 /// Generates a special objective and adds it to the objective list.
 /datum/antagonist/malf_ai/proc/forge_special_objective()
-	var/special_pick = rand(1,4)
+	// SKYRAT EDIT START
+	var/special_pick = rand(1,2)
```

Comment:
Just change it into a `rand(3, 4)` instead, that ends up being a one-line change.

Reply:
You know, this is one of those things that _feels_ like it should've been the obvious solution at the time, but this is what we have code review for